log/grpc.go

```diff
@@ -218,6 +243,14 @@ func WithDisableCallID() GRPCLogOption {
 	}
 }
 
+// WithCallLogFunc returns a HTTP middleware option that configures the logger to use
+// the given log function instead of log.Print() as default.
```

Comment:
Should this comment be referring to HTTP middleware since this is actually for a gRPC interceptor?

Reply:
#528